common/CMakeLists.txt

```diff
@@ -278,7 +278,7 @@ add_dependencies(tests crypto_test)
 find_package(PkgConfig REQUIRED)
 find_package(Boost 1.35 REQUIRED)
 pkg_check_modules(dbus REQUIRED dbus-1)
-add_library(common_dbus STATIC dbus/dbus.cpp dbus/platform/asio_libdbus/dbus_client.cpp)
+add_library(common_dbus STATIC dbus/dbus.cpp dbus/platform/asio_libdbus/dbus.cpp)
```

Comment:
> Also rename dbus_client.cpp to dbus.cpp because it now contains non-client code. At the same time it's not big enough for splitting it into multiple files to be worth the extra complexity.

It might be worth it, because the server and the client will not be used by the same binaries. Mender-auth will only use the server, and mender-update will only use the client, if I'm not mistaken. Of course this doesn't *require* us to split it, but might help reduce code size if we can keep them independent. Any thoughts on this?

Reply:
Lot of the code belongs to `DBusPeer`. So we would have to split things in 3 files, some functions could no longer be `static` and we would have to add some internal header file. With the client and the server being much smaller parts on top of a bigger pile shared by both. I'd suggest we keep things simple for now and we will see if code size optimization is needed and worth it here.